File: CCC Java/ccc14j1_triangle_times/TriangleTimes.java
//DMOJ - Triangle Times
import java.util.Scanner;
//Enter Angles of Triangle that add up to 180

public class TriangleTimes
{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
        
        try{

            int d1 = sc.nextInt();
            int d2 = sc.nextInt();
            int d3 = sc.nextInt();
            
            if((d1==60)&&(d2==60)&&(d3==60))
            {
                System.out.println("Equilateral");//Equalateral
                
            }else if(d1 + d2 + d3 == 180){
                
                if((d1==d2)||(d1==d3)||(d2==d3)){
                    System.out.println("Isosceles");//Isosceles
                    
                }else if((d1!=d2)&&(d1!=d3)&&(d2!=d3)){
                    System.out.println("Scalene");//Scalene
                }
            }
            else
            {
                System.out.println("Error");
            }
        }
        finally{
            sc.close();
        }
	}
}

File: CCC Python/2014/Junior 1/triangleTimes.py
#Asks for the 3 degrees of the triangle
d1 = int(input())	
d2 = int(input())
d3 = int(input())


if((d1 == 60) and (d2 == 60) and (d3 == 60)): 	#if all angles are 60, the triangle is equilateral
    print ("Equilateral")
    
elif(d1 + d2 + d3 == 180):
    if((d1 == d2) or (d1 == d3) or (d2 == d3)):	 #if any two of the 3 angles are equal, it’s Isosceles
        print ("Isosceles")
  
    elif((d1!=d2) and (d1!=d3) and (d2!=d3)):  #if none of the degrees are equal, the triangle is Scalene
        print ("Scalene")
else:
    print ("Error")
